config/mtl.toml

```diff
@@ -630,3 +630,22 @@ count = 25
 	pin = [0, 0, 0xfeef, 0xf, 0xf, 0x45ff, 1, 0, 0xfeef, 0xf, 0xf, 0x1ff]
 	# mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
 	mod_cfg = [0, 0, 0, 0, 4096, 1000000, 128, 128, 0, 0, 0]
+
+	# dts codec module config
+	[[module.entry]]
+	name = "DTS"
```

Comment:
is dts PORed for mtl chrome book? also, how to test DTS codec functionality?

Reply:
@btian1 , not POR, but customer demands. I am working with DTS engineer @joechengxperi , we are preparing to submit the relevant DTS topology conf as well. The DTS EQIIR function is working based on them. 
One question:
Currently we set the default value on mod_cfg. Do you or someone know how to tuning the mod_cfg values ? Is there a tool to do that?

	mod_cfg = [0, 0, 0, 0, 296, 5000000, 384, 384, 0, 5000, 0]

Reply:
I don't know how to tunning this, maybe @kv2019i @singalsu  know this part.
one more question is: we already have component EQIIR, what's the difference between existing EQIIR and DTS EQIIR?

Reply:
To clarify that, we use existing EQ IIR + DTS currently, not DTS EQIIR.

Reply:
That's nice to hear, don't hesitate to suggest improvements for IIR component or library if you come up with ideas.